docs/readme.md

```diff
@@ -0,0 +1,56 @@
+# WordPress Term Debt Consolidator
+
+Term Debt Consolidator can look at your WordPress site's tags to suggest areas for consolidation and deletion and help you identify ways to improve your use of WordPress' taxonomy functionality.
+
+Keep in mind that this process is imperfect and may make nonsensical suggestions from time to time. It's up to you to review the suggestions and approve or deny them.
+
+## Installation
+
+Term Debt Consolidator can be installed like any other WordPress plugin.
+
+1. Install the plugin through the WordPress plugins screen directly, or download the zip file from this Github repo and install manually to your WordPress site.
```

Comment:
let's be more specific here, most people will understand what that means, but I'd like the steps to be a bit more explicit